test/tests/markdown.test.ts

```diff
@@ -1,29 +0,0 @@
-import { test } from '../test-utils';
-
-const files = import.meta.glob('/test/fixtures/markdown/*/*', {
-	assert: { type: 'raw' },
-});
-
-// *** MARKDOWN ***
-// test(
-//   'can format an Astro file containing an Astro file embedded in a codeblock',
-//   files,
-//   'markdown/embedded-in-markdown',
-//   { mode: 'markdown' }
-// );
```

Comment:
This was commented out due to another bug in the formatting. I believe we should support this.